### PR TITLE
Home stg

### PR DESCRIPTION
**Pull request by vitalii-toderian**

### CODE REVIEW

File: src/components/FooterComponent/FooterComponent.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import React, {Component} from 'react';
+import './FooterComponent.scss'
+
+class FooterComponent extends Component {
+    render() {
+        return (
+            <div className="container-fluid">
```
Comment (vste23) [inline src/components/FooterComponent/FooterComponent.js]: no need to use container-fluid and row just to have full width container, div by default already takes full width

File: src/components/FooterComponent/FooterComponent.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                <div className="row footer-container footer-dimensions">
```
Comment (vste23) [inline src/components/FooterComponent/FooterComponent.js]: no need row here

File: src/pages/home/HomeQuestions/QuestionSection.scss [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    .msg-text {
+        margin-bottom: 30px;
+        font-size: 24px;
+    }
+
+    .questions-header {
+        margin-bottom: 20px;
+    }
+
+    @media (min-width: 360px) {
```
Comment (vste23) [inline src/pages/home/HomeQuestions/QuestionSection.scss]: empty styles

**Review by vste23 — CHANGES_REQUESTED** [review]

**Comment by vitalii-toderian** [comment]
image
image
image
desktop empty space at the bottom will be fixed in next commits

File: src/pages/home/HomeQuestions/QuestionSection.scss [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+.question-section::before {
+    content: "";
+    position: absolute;
+    z-index: 0;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-image: url("./questions_bkg_mobile_old.png");
```
Comment (vste23) [inline src/pages/home/HomeQuestions/QuestionSection.scss:11]: add also
`    background-size: cover;
    background-position: center center;
    background-repeat: no-repeat;`

File: src/pages/home/HomeQuestions/QuestionSection.scss [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+    filter: opacity(0.1);
+    object-fit: fill;
+}
+
+@media (min-width: 720px) {
+    .question-section::before {
+        background-image: url("./questions_bkg_tablet.png");
```
Comment (vste23) [inline src/pages/home/HomeQuestions/QuestionSection.scss:21]: add also 
`    background-size: cover;
    background-position: center center;
    background-repeat: no-repeat;`

File: src/pages/home/HomeQuestions/QuestionSection.scss [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+    }
+}
+
+@media (min-width: 992px) {
+    .question-section::before {
+        background-image: url("./questions_bkg_desktop.png");
```
Comment (vste23) [inline src/pages/home/HomeQuestions/QuestionSection.scss:27]: add also 
`    background-size: cover;
    background-position: center center;
    background-repeat: no-repeat;`

**Review by vste23 — CHANGES_REQUESTED** [review]

**Pull request merged**